enhancements/etcd/cluster-etcd-operator.md

```diff
@@ -490,7 +490,23 @@ items:
 
 ### Upgrade / Downgrade Strategy
 
-TODO
+#### Upgrade from 4.3 to 4.4
+
+quorum-guard: currently quorum-guard is a deployment in the `machine-config-operations` namespace the bash logic used
```

Comment:
Add a section above in the main design details that explains what this does, why we have it, and how it works.

Reply:
> Add a section above in the main design details that explains what this does, why we have it, and how it works.

That is valid for 4.4 I mean.